### PR TITLE
[FEATURE] improved, working example of type = user

### PR DESCRIPTION
**Pull request by linawolf**
This PR also references https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/pull/1242

### CODE REVIEW

File: Documentation/ColumnsConfig/Type/User.rst [hunk 1]
```diff
@@ -45,51 +45,142 @@ The `user` field can be useful, if:
...
+
+            $attributes['placeholder'] = 'Enter special value for user "'.htmlspecialchars(trim($row['username'])).
+               '" in size '. $size;
+            $classes = [
+               'form-control',
+               't3js-formengine-textarea',
+               'formengine-textarea',
+            ];
+            $itemValue = $parameterArray['itemFormElValue'];
+            $attributes['class'] = implode(' ', $classes);
```
Comment (mbrodala) [inline Documentation/ColumnsConfig/Type/User.rst:131]: Is there a specific reason why not all of this is done before / during `$attributes` initialization?

File: Documentation/ColumnsConfig/Type/User.rst [hunk 2]
```diff
@@ -45,51 +45,142 @@ The `user` field can be useful, if:
...
+            $html[] =         '<div class="form-control-wrap">';
+            $html[] =            '<input type="text" value="' . htmlspecialchars($itemValue, ENT_QUOTES) . '" ';
+            $html[]=               GeneralUtility::implodeAttributes($attributes, true);
+            $html[]=            ' />';
+            $html[] =         '</div>';
+            $html[] =      '</div>';
+            $html[] =   '</div>';
+            $html[] = '</div>';
+            $resultArray['html'] = implode(LF, $html);
+            return $resultArray;
```
Comment (mbrodala) [inline Documentation/ColumnsConfig/Type/User.rst:147]: We should always have an empty line before `return`.

File: Documentation/ColumnsConfig/Type/User.rst [hunk 3]
```diff
@@ -45,51 +45,142 @@ The `user` field can be useful, if:
...
+         }
+      }
+
+      .. important::
```
Comment (mbrodala) [inline Documentation/ColumnsConfig/Type/User.rst:151]: I'd use `attention` instead here.

**Review by mbrodala — CHANGES_REQUESTED** [review]

File: Documentation/ColumnsConfig/Type/User.rst (hunk 1 above)
Comment (mbrodala) [reply Documentation/ColumnsConfig/Type/User.rst:131]: @linawolf Can you have a look at this?

Comment (linawolf) [reply Documentation/ColumnsConfig/Type/User.rst]: The code is taken from the examples extension and is proven as working. You can make a PR there if you want to change it.

**Pull request merged**